### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by jer3m01** [rome/tools#1252] [General]
Keep track and make a list of all the things that have to be done for rome.

**Comment by ematipico** [comment]
Sebastian will focus on what's in the funding campaign and probably will work on that part. Is there anything else that you want to add to help?  You mentioned the bundler on discord and we have another discussion about it.

**Reply by jer3m01** [reply]
Yeah, I'd love to help on the testing library and the documentation generation, Although I only have a concept of how each of them work, not quite sure how they are supposed to be implemented.

**Comment by sebmck** [comment]
Just wanted to note I'm going to draft a longer reply with my thoughts <strike>tonight</strike> tomorrow! Just didn't want you to think I missed this. Thanks @jer3m01 for getting this conversation started!
